specs/001-design-and-implement/quickstart.md

```
@@ -0,0 +1,401 @@
+    // Custom build logic
+    generator.compile_to_assembly("src/main.jsav", &target)
+        .expect("Assembly generation failed");
+}
+```
+
+## Troubleshooting
+
+### Common Issues and Solutions
+
+**Issue**: `UnsupportedInstruction` error
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)